bundles/org.openhab.ui/web/src/pages/settings/things/inbox/inbox-list.vue

```diff
@@ -33,6 +36,7 @@
       <div class="right" v-if="$theme.md">
         <f7-link v-show="selectedItems.length" icon-md="material:delete" icon-color="white" @click="confirmActionOnSelection('delete')" />
         <f7-link v-show="selectedItems.length" icon-md="material:visibility_off" icon-color="white" @click="confirmActionOnSelection('ignore')" />
+        <f7-link v-show="selectedItems.length" icon-md="material:visibility_off" icon-color="white" @click="confirmActionOnSelection('unignore')" />
```

Comment:
The unignore icon is the same as the ignore icon in the MD theme... that's not ideal.

Reply:
you are absolutely right, I overlooked this line. Should be 'icon-md="material:visibility"'
I will fix that.

Reply:
```suggestion
        <f7-link v-show="selectedItems.length" icon-md="material:visibility" icon-color="white" @click="confirmActionOnSelection('unignore')" />
```